package.json

```diff
@@ -31,7 +31,7 @@
   },
   "wireit": {
     "bidi-types": {
-      "command": "tools/generate-bidi-types.sh",
+      "command": "./tools/generate-bidi-types.sh",
```

Comment:
There's no need for `./` here, as it was previously. Same for other commands under `tools/`.

Reply:
Other places use this as well. I'm not sure if we need to change it right now. I'm going to add Windows CI so we can validate it then.